ngraph/core/include/ngraph/function.hpp

```diff
@@ -138,6 +149,25 @@ namespace ngraph
         bool evaluate(const HostTensorVector& output_tensors,
                       const HostTensorVector& input_tensors) const;
 
+        /// \brief Return a list of function's sinks.
+        const SinkVector& get_sinks() const { return m_sinks; }
+        /// \brief Add new sink nodes to the list.
+        /// \param sinks new sink nodes
+        void add_sinks(const SinkVector& sinks);
```

Comment:
from my point of view we must have two methods `add_sink` and `ass_sinks`

Reply:
the same about result

Reply:
possibly 2 methods are useful, but currently we can use 1 and look if we will often use add_sinks/add_results for 1 sink/result, then we can add  the second one.